pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+//nolint:unused
```

Comment:
Since we skip the whole file, do we still need this?

Reply:
Thanks, this was a leftover from up above when I was trying to figure out why this file wasn't being skipped properly. Removed.